CSC8503/LevelManager.cpp

```diff
@@ -414,6 +415,7 @@ void LevelManager::LoadItems(const std::vector<Vector3>& itemPositions, const bo
 		}
 		else {
 			AddPickupToWorld(itemPositions[i], mInventoryBuffSystemClassPtr, isMultiplayer);
+			//AddPointObjectToWorld(itemPositions[i]);
```

Comment:
Is there a reason we're leaving this in? Remove comments if you're not using them and you can re-add them in later, or add a to-do for it.

Reply:
Correct me if I'm mistaken, but I don't think we've got the positions for the point objects in the level manager yet, so I temporarily used the pickup positions.